database/report.py

```diff
@@ -1,7 +1,7 @@
 imaginative = {
-    "zipcode": 11111,
-    "credit_score": 695,
-    "salary": 55000,
+    "zipcode": 60654,
+    "credit_score": 617,
+    "salary": 3000,
```

Comment:
These were an awesome exercise of edge-testing and sad-pathing for this code.

Reply:
YES! I can't believe how long it took for me to find out that our code was breaking because we were using a "fake" zipcode!